app/store/LayerCategoriesStore.js

```diff
@@ -0,0 +1,26 @@
+import Store from 'fluxible/addons/BaseStore';
+
+class LayerCategoriesStore extends Store {
+  static storeName = 'LayerCategoriesStore';
+
+  layerCategories = [];
+
+  constructor(dispatcher) {
+    super(dispatcher);
+
+    const context = dispatcher.getContext();
+
+    fetch(context.config.layerCategoriesUrl)
```

Comment:
This will break for other stadtnavi instances, no? I think we should fall back to a minimal default config here. Let's discuss how it should like

Reply:
No this won't break other instances, because they are not using external provided layer config at all.